packages/flutter_tools/lib/src/commands/build_apk.dart

```diff
@@ -137,12 +137,26 @@ class BuildApkCommand extends BuildSubCommand {
     validateBuild(androidBuildInfo);
     displayNullSafetyMode(androidBuildInfo.buildInfo);
     globals.terminal.usesTerminalUi = true;
+    final FlutterProject project = FlutterProject.current();
     await androidBuilder?.buildApk(
-      project: FlutterProject.current(),
+      project: project,
       target: targetFile,
       androidBuildInfo: androidBuildInfo,
       configOnly: configOnly,
     );
+
+    // When an app is successfully built, record to analytics whether Impeller
+    // is enabled or disabled. Note that 'computeImpellerEnabled' will default
+    // to false if not enabled explicitly in the manifest.
+    final bool impellerEnabled = project.android.computeImpellerEnabled();
+    final String buildLabel = impellerEnabled
+          ? 'manifest-impeller-enabled'
+          : 'manifest-impeller-disabled';
+    globals.analytics.send(Event.flutterBuildInfo(
+      label: buildLabel,
```

Comment:
Does the 2 values for label get put in the same bucket so in analytics you'll get a pie chart showing the percentage of one versus the other?  As opposed to being disjoint tallies?

Reply:
Right, I talked to @zanderso and @jonahwilliams about this and emulated the iOS behavior for that reason.